src/frontend/src/pages/PPQPage/PPQPage.tsx

```diff
@@ -1,35 +1,92 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import {
+  faChevronRight,
+  faExclamationTriangle,
+} from '@fortawesome/free-solid-svg-icons';
 
 import ppqImg from '../../assets/ppq_homepage.svg';
 
 import { Link } from 'react-router-dom';
+import messages from './messages';
 
 interface IComponentProps {
   enablePiaIntakeForm: boolean;
 }
 function PPQLandingPage(props: IComponentProps) {
-  // TODO For ticket 402 use pia value to decide render pia intake or not
   const { enablePiaIntakeForm } = props;
-  return (
+  return enablePiaIntakeForm ? (
     <div className="bcgovPageContainer background">
-      <div className="ppq-section results-wrapper">
-        <div className="row what-is-a-ppq">
+      <div className="get-started-section results-wrapper">
+        <div className="row">
           <div className="col-md-6">
-            <h2>PIA Pathway Questionnaire</h2>
+            <h2 className="form-heading">{messages.PIAIntakeHeading.en}</h2>
```

Comment:
What does this class do that the h2 tag cannot?

Reply:
text weight is 700